spark/src/main/scala/org/apache/spark/sql/delta/PreprocessTableMerge.scala

```diff
@@ -98,7 +95,8 @@ case class PreprocessTableMerge(override val conf: SQLConf)
       throw DeltaErrors.operationOnTempViewWithGenerateColsNotSupported("MERGE INTO")
     }
     // Additional columns with default expressions.
-    var additionalColumns = Seq[StructField]()
+    val additionalColumns = IdentityColumn.getIdentityColumns(
+      tahoeFileIndex.snapshotAtAnalysis.metadata.schema)
```

Comment:
Can you rename `additionalColumns` to `identityColumns`? It's only ever passed in the code where we need to pass an indentity column and there are no other usage AFAICT.